### PR TITLE
Escaping inner quotes

### PR DESCRIPTION
**Pull request by viralkachhadiya**
it's for #77436

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ ] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by viralkachhadiya** [comment]
@goderbauer 
Please checkout it.

**Comment by goderbauer** [comment]
Looks like there are still some failures in example code: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8852521594922623424/+/steps/prepare_environment/0/steps/Analyze/0/steps/analyze/0/stdout

**Comment by viralkachhadiya** [comment]
> Looks like there are still some failures in example code: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8852521594922623424/+/steps/prepare_environment/0/steps/Analyze/0/steps/analyze/0/stdout

Okay,thanks

File: dev/bots/test/analyze_sample_code_test.dart [hunk 1]
```diff
@@ -20,23 +20,23 @@ void main() {
...
       '>>> info: Prefer const over final for declarations (prefer_const_declarations)',
-      'known_broken_documentation.dart:111:23: final String? bar = \'Hello\';',
+      "known_broken_documentation.dart:111:23: final String? bar = 'Hello';",
       '>>> info: Use a non-nullable type for a final variable initialized with a non-nullable value (unnecessary_nullable_for_final_variable_declarations)',
       'known_broken_documentation.dart:112:9: final int foo = null;',
       '>>> info: Prefer const over final for declarations (prefer_const_declarations)',
       'known_broken_documentation.dart:112:25: final int foo = null;',
-      '>>> error: A value of type \'Null\' can\'t be assigned to a variable of type \'int\' (invalid_assignment)',
+      ">>> error: A value of type 'Null' can't be assigned to a variable of type 'int' (invalid_assignment)",
       '',
```
Comment (Hixie) [inline dev/bots/test/analyze_sample_code_test.dart:40]: Having this code flip between double quotes and single quotes like this is really confusing.
Are we sure we want this lint enabled?

Comment (gspencergoog) [reply dev/bots/test/analyze_sample_code_test.dart:40]: I actually find it much easier to read without the escapes, and I hardly notice the difference in the double/single quotes.

Comment (goderbauer) [reply dev/bots/test/analyze_sample_code_test.dart:40]: I second Greg. I find it makes the actual string (the thing I care about) easier to read.

Comment (viralkachhadiya) [reply dev/bots/test/analyze_sample_code_test.dart:40]: @goderbauer what you think about it?

Am i continue working on it??

**Comment by viralkachhadiya** [comment]
@goderbauer please help me

File: dev/benchmarks/complex_layout/test_driver/scroll_perf_test.dart [hunk 2]
```diff
@@ -50,7 +50,6 @@ void main() {
       });
 
       final TimelineSummary summary = TimelineSummary.summarize(timeline);
-      await summary.writeSummaryToFile(summaryName, pretty: true);
```
Comment (goderbauer) [inline dev/benchmarks/complex_layout/test_driver/scroll_perf_test.dart:53]: Why the removal of these lines?

Comment (goderbauer) [reply dev/benchmarks/complex_layout/test_driver/scroll_perf_test.dart:53]: here and elsewhere in the PR.

**Review by goderbauer — COMMENTED** [review]
Can you resolve the merge conflicts?

**Pull request closed without merging**